lib/bridge/strain.rb

```diff
@@ -36,6 +36,21 @@ def self.minors
       [Club, Diamond]
     end
 
+    @mapping = {}
+    %w(Clubs Diamonds Hearts Spades NoTrumps
```

Comment:
I find horizontal alignment pleasing if there are multiple lines with similar structure. And my editor has a built-in horizontal alignment utility, so I'm rarely doing it manually (except when adding a line to an already-aligned block).

However, I *do* lean toward meta-programming or similar tactics to compact those lists into 1 line when possible. 

Here is some code from my current task:
```
  let(:serial_number) { SecureRandom.uuid }
  let(:store) { create(:store) }
  let(:position) { create(:position, store: store) }
  let(:security_device) { create(:security_device, store: store, position: position) }
  let(:status_id) { SecureRandom.uuid }
```
`C-u M-x align-regexp` and:

```
  let(:serial_number)   { SecureRandom.uuid        }
  let(:store)           { create(:store)           }
  let(:position)        { create(:position,        store: store) }
  let(:security_device) { create(:security_device, store: store, position: position) }
  let(:status_id)       { SecureRandom.uuid        }
```